test/deploy-pipeline-lite.sh

```diff
@@ -47,9 +49,19 @@ if [ -z "$KFP_DEPLOY_RELEASE" ]; then
   popd
 
   pushd ${KFP_MANIFEST_DIR}/dev
-  echo imagePrefix=${GCR_IMAGE_BASE_DIR} > params.env
-  echo imageTag=${GCR_IMAGE_TAG} >> params.env  
-  cat params.env
+  # This is the recommended approach to do this.
+  # reference: https://github.com/kubernetes-sigs/kustomize/blob/master/docs/eschewedFeatures.md#build-time-side-effects-from-cli-args-or-env-variables
```

Comment:
I'm wondering whether the change increased the complexity actually.
Feel free to go, just feel "kustomize edit set image" didn't well solve the problem

Reply:
Complexity for test scripts seem trivial to me. The important part is it left the flexibility of overriding images to users by kustomize.